.build-tools/pkg/metadataschema/builtin-authentication-profiles.go

```diff
@@ -20,6 +20,68 @@ import (
 // ParseBuiltinAuthenticationProfile returns an AuthenticationProfile(s) from a given BuiltinAuthenticationProfile.
 func ParseBuiltinAuthenticationProfile(bi BuiltinAuthenticationProfile) ([]AuthenticationProfile, error) {
 	switch bi.Name {
+	case "aws":
+		return []AuthenticationProfile{
+			{
+				Title:       "AWS: Access Key ID and Secret Access Key Dapr Metadata",
+				Description: "Authenticate using an Access Key ID and Secret Access Key included in the metadata",
+				Metadata: []Metadata{
+					{
+						Name:        "AWS_ACCESS_KEY_ID",
+						Required:    true,
+						Sensitive:   true,
+						Description: "AWS access key associated with an IAM account",
+						Example:     `"AKIAIOSFODNN7EXAMPLE"`,
+					},
+					{
+						Name:        "AWS_SECRET_ACCESS_KEY",
+						Required:    true,
+						Sensitive:   true,
+						Description: "The secret key associated with the access key",
+						Example:     `"wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"`,
+					},
+				},
+			},
+			{
+				Title:       "AWS: Credentials from Environment Variables",
+				Description: "Use AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY",
+				Metadata: []Metadata{
```

Comment:
These are env vars so should NOT be in the metadata. This is an empty slice here